lib/_stream_writable.js

```diff
@@ -307,7 +307,6 @@ Writable.prototype.uncork = function() {
 
     if (!state.writing &&
         !state.corked &&
-        !state.finished &&
```

Comment:
why is this unused?

Reply:
`state.finished` should equal `false` if `state.bufferedRequest` isn't `null`.